### PR TITLE
fix #60338 m.vk.com

### LINKED ISSUE #60338: m.vk.com
### Issue URL (Ads)
 [https://m.vk.com/overlordsdub](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fm.vk.com%2Foverlordsdub)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 9
AdGuard product: | AdGuard for Android v3.5 nightly 4 (29)
Browser: | Chrome
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS requests blocking: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#60338

### CODE REVIEW

File: MobileFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59614
 mensxp.com#?#.list-card > li:not(.track_ga):has(> div[class*=" "] > div > div > a[onclick])
+! https://github.com/AdguardTeam/AdguardFilters/issues/60338
+m.vk.com#?#.wall_posts > div.wall_item:has(> div.wi_body > div.ads_mark)
```
Comment (Alex-302) [inline MobileFilter/sections/css_extended.txt:7]: On my end - element between `.wall_posts` and `div.wall_item`
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: MobileFilter/sections/css_extended.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply MobileFilter/sections/css_extended.txt:7]: I checked my rule, and it worked for me. I'll check again.

Comment (Alex-302) [reply MobileFilter/sections/css_extended.txt:7]: just make `.wall_posts div.wall_item`

Comment (Sergey-Lyapin) [reply MobileFilter/sections/css_extended.txt:7]: Ok
Screenshot 2020-08-01 at 22 34 58

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**